src/stan_math_backend/Lower_expr.ml

```diff
@@ -102,7 +102,7 @@ let promote_adtype =
 
 let suffix_args = function
   | Fun_kind.FnRng -> ["base_rng__"]
-  | FnTarget -> ["lp__"; "lp_accum__"]
+  | FnTarget | FnJacobian -> ["lp__"; "lp_accum__"]
```

Comment:
As far as I know (which isn't much) `lp__` is [only a vestige](https://mc-stan.org/docs/reference-manual/removals.html#lp__-variable) and no current user code can touch it. I don't know if `FnTarget` needs to have a backwards-compatible argument list but `FnJacobian` could be simplified.
```ocaml
 | FnJacobian -> ["lp_accum__"]
```

Reply:
I believe it is still used by the constraint code, so I think if we ever expose the built-in constraints through this functionality it will need it.